package.json

```diff
@@ -27,18 +27,20 @@
     "write-changelog": "./node_modules/.bin/conventional-changelog -i CHANGELOG.md -s -p angular"
   },
   "dependencies": {
-    "angular2": "2.0.0-beta.3",
-    "es6-promise": "3.0.2",
+    "angular2": "2.0.0-beta.7",
+    "es6-promise": "^3.0.2",
```

Comment:
It would be better to use exact versions without '^'
Cause as showed my experience not all package provider know what the semantic versioning is :)

In order to install exact version via npm you can simply do following
npm install (--save|--save-dev) ${packageName} `--save-exact`